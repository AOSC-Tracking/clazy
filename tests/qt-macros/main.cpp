#ifdef Q_OS_WIN // Warning: testing before including qglobal.h
#endif

#if defined(Q_OS_WIN) // Warning: testing before including qglobal.h
#endif

#include <QtCore/QString>

#if defined(Q_OS_WINDOWS) // Warning, should be Q_OS_WIN
#endif

#ifdef Q_OS_WINDOWS // Warning, should be Q_OS_WIN
#endif

#ifdef Q_OS_WIN // OK
#endif

#if defined(Q_OS_WIN) // OK
#endif
