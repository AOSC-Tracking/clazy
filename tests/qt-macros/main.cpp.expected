qt-macros/main.cpp:1:2: warning: Include qglobal.h before testing Q_OS_ macros [-Wclazy-qt-macros]
qt-macros/main.cpp:4:5: warning: Include qglobal.h before testing Q_OS_ macros [-Wclazy-qt-macros]
qt-macros/main.cpp:9:5: warning: Q_OS_WINDOWS is wrong, use Q_OS_WIN instead [-Wclazy-qt-macros]
qt-macros/main.cpp:12:2: warning: Q_OS_WINDOWS is wrong, use Q_OS_WIN instead [-Wclazy-qt-macros]
